### PR TITLE
Added the CELTIC lawset to the game

### PR DESCRIPTION
**Pull request by sonime**
This PR adds a new lawset, dubbed "CELTIC" after the player who made it. It has only one law:

"Be someone the crew can look up to and respect."

It adds the object, the research data, and the module itself to the AI upload on Box.

### CODE REVIEW

File: html/changelogs/DeadPoolJ.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author: DeadPoolJ
+changes:
+	-rscadd: Adds the CELTIC core module and board, with the following law: "Be someone the crew can look up to and respect."
```
Comment (unid15) [inline html/changelogs/DeadPoolJ.yml]: Two spaces here, not tabs. Also "Added" sounds a bit better for the changelog

Comment (sonime) [inline html/changelogs/DeadPoolJ.yml]: Now how do I update the file without making a new PR?

This is the second PR I've done and I am new to Git.

Comment (unid15) [inline html/changelogs/DeadPoolJ.yml]: Just change the file and create a new commit

It'll appear here

**Comment by sonime** [comment]
Ok fixed it thanks.

**Comment by sonime** [comment]
Why is the build failing?

**Comment by ghost** [comment]
The changelog's markup is wrong somehow, are you sure you are putting spaces where they belong ?

**Comment by sonime** [comment]
I'm not sure. Two spaces in front of rscadd, no other spaces.

**Comment by unid15** [comment]
Could be an issue with the quotation marks ""

**Comment by sonime** [comment]
I still don't get why the build is failing. I did the map smoothing thing I was supposed to.

**Comment by sonime** [comment]
NOW IT WORKS

**Comment by Konater** [comment]
2: Rangers fans are not part of the crew

**Comment by Imagebard** [comment]
>On Box

What about Taxi, Meta, And Def?

**Comment by Imagebard** [comment]
Wait, better idea.
Remove it form Meta and Defficiency so I can brag about how Boxstation and Taxistation are worse than the other two because they don't have shitty lawsets installed.

**Comment by sonime** [comment]
TOO LATE FAG ITS IN

**Comment by NigglyWiggly** [comment]
Not if it doesn't get metrged

**Comment by PJB3005** [comment]
:+1: to this so much.

I had this lawset once as AI, it was fun.

**Comment by d3athrow** [comment]
somebody got a permaban for this lawset once, plasma flooding does not make you somebody others can look up to or respect

**Comment by sonime** [comment]
Each time I've seen it uploaded aside from that, it works really well. I've had good response to it uploading it myself and talking with the silicons after they were placed on something different/subverted.

The plasmaflooding guy was being a shit, that's all there is to it.

**Comment by clusterfack** [comment]
Alternate title: Welcome to git, enjoy your stay

**Comment by sonime** [comment]
?

**Comment by icantthinkofanameritenow** [comment]
Sonime, this can be replicated perfectly with a single Freeform Core module. This is just stupid and wasteful, and it bloats the codebase even more.

**Comment by sonime** [comment]
Eh, I like having it as something readily accessible ingame and there to get people to try out different lawsets.

**Comment by icantthinkofanameritenow** [comment]
@sonime 
Freeform core modules are plenty accessible, what are you talking about? Is it that hard to copypaste the law's text in?

**Comment by ihadtoregisterforthis** [comment]
Same thing could be said for every lawset

**Comment by sonime** [comment]
Image of Dracula

**Pull request merged**